schemas/telemetry/frecency-update/frecency-update.4.schema.parquetmr.txt

```diff
@@ -13,6 +13,7 @@ message frecency-update {
     required int32 rank_selected;
     required int32 num_suggestions_displayed;
     required binary study_variation;
+    required int32 Timestamp;
```

Comment:
This is a long and needs to be int64. I'd also put it as a the first field, before clientId

Reply:
I changed it to a long now. In the last review, Anthony told me to order the fields alphabetical. Is Timestamp an exception to this or should I leave it completely alphabetically sorted?

Reply:
For parquet? I suspect he meant for the json schema, since a json object isn't ordered, but the parquet schema will determine the actual column order that's output

Reply:
Oh, I wasn't aware of that. `Timestamp` is the first element in `payload` now.